train_trim: 25
